### DISCUSSION TITLE
What green vegetable is apparently one of 🇧🇷's most popular 🍕 toppings?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#704] [Galaxy Trivia] [answered]
**Bonus:** What is the 🇨🇷 people's beloved pizza topping?

**Comment by peterbaccaro** [comment]
**Green peas** are one of the most popular pizza toppings in Brazil.
**Coconut** is a popular pizza topping for the Costa Rican people.

**Comment by thuongtruong1009** [comment, marked as answer]
The green vegetable that is apparently one of Brazil's most popular pizza toppings is hearts of palm, known as "palmito" in Portuguese. Hearts of palm are harvested from the inner core of certain types of palm trees and have a mild, slightly sweet flavor and tender texture.

In Costa Rica, one of the most beloved pizza toppings is "chorizo", which is a type of spicy sausage that is popular in Spanish and Latin American cuisine. Other popular toppings in Costa Rica include "tocino" (bacon), "champiñones" (mushrooms), "jalapeños" (pickled hot peppers), and "piña" (pineapple).

**Reply by ParanoidUser** [reply]
Fascinating, thanks! 👍
